debian/control

```diff
@@ -0,0 +1,27 @@
+Source: jq
+Section: utils
+Priority: optional
+Maintainer: Simon Elsbrock <simon@iodev.org>
+Build-Depends: debhelper (>= 9), dh-autoreconf, flex, bison, valgrind [amd64 i386], rake, ruby-ronn, libonig-dev
+Standards-Version: 3.9.6
+Homepage: https://github.com/stedolan/jq
+Vcs-Git: git://anonscm.debian.org/users/else-guest/jq.git
+Vcs-Browser: http://anonscm.debian.org/gitweb/?p=users/else-guest/jq.git
+
+Package: jq
+Architecture: any
+Multi-Arch: foreign
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: lightweight and flexible command-line JSON processor
+ jq is like sed for JSON data – you can use it to slice
+ and filter and map and transform structured data with
+ the same ease that sed, awk, grep and friends let you
+ play with text.
+ .
+ It is written in portable C, and it has zero runtime
```

Comment:
Oh, we should probably update this.  With Oniguruma, jq now has 1 run-time dependency.